community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -13,6 +13,15 @@ Applicants should explore employer-sponsored funding options as their first sour
 Additionally, 25% of the community fund is allocated specifically to support underrepresented groups.
 Funding decisions are determined by a combination of need and potential impact.
 
+## Transparency
+
+In **2023**, 26 funding requests were made by 22 contributors for a total requested sum of USD 47,500 with a maximum of 2 requests per contributor.
+Requested sums per trip ranged from $750 to $5,750, with a median of $1,525 and an average of $1,825.
+Requested sums per individuals ranged from $1,000 to $7,350, with a median of $1,950 and an average of $2,150.
+All requests were granted.
+
+In 2024, USD 60,000 was allocated to the fund.
```

Comment:
```suggestion
In 2024, USD $60,000 was allocated to the fund.
```

Reply:
There's a USD prefix on the first one so it's clear these are USD and not dollars from elsewhere.

Reply:
There’s 4 dollar amounts in line 20 that just use $.

Reply:
it looks weird to me to ever see a USD amount without a dollar sign, to be clear :-)